### ISSUE TITLE
Menu bar goes onto two lines unnecessarily for certain screen widths.

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#250]
There would be space for all components if menu bar could be off centre.

See #227

**Comment by hyanwong** [comment]
Also, with some recent css changes in (I think) 620c2bd2711d298a0b02ac94dde4693a93b1cb32 the menubar is no longer centred if it wraps onto a new line

**Comment by hyanwong** [comment]
Hi @lentinj - just labelling this as a possible outsource, if only to fix the now non-centralised menu when the page shrinks to narrow widths. Ta!

**Comment by lentinj** [comment]
I'll have a look, there's not an easy CSS way to express what @jrosindell means for the overall ticket (if I'm remembering this correctly), but the not-centered part is probably much easier to work out.

**Comment by jrosindell** [comment]
@lentinj, yes I remember, we only want it to be centered then we can close the issue - I think we're not worried about the original problem but leave the issue open for that to be fixed later if we decide it's worth it.

**Comment by jrosindell** [comment]
Note that the CSS for the front page is now stored in a different way and it's this that caused the problem....  content.scss is for everything that's used across the whole site content_index.scss is for the front page elements not used anywhere else and content_pages.scss is for the content that appears on other pages, but never on the index page.

**Comment by lentinj** [comment]
FWIW it looked to me to be a UI-kit upgrade that caused probems, not the re-jigging.

Regardless, centring is sorted with the above.

**Comment by hyanwong** [comment]
Great, thanks. Would have taken me a while to spot that.
